### DISCUSSION TITLE
Keep playback synced across all windows by default

### DISCUSSION THREAD

**Discussion by loheiman** [f1multiviewer/issue-tracker#37] [Feature requests]
I think the default behavior for controlling playback should control all windows/streams. I think there are pretty limited use cases where folks would not want the streams synced. Now sometimes it appears that there is a small discrepancy between streams and so the ability to add an offset/delay is definitely helpful.

**Comment by f1multiviewer** [comment]
Do you mean for live or replays?

**Reply by loheiman** [reply]
For replays I guess. I haven't used the app live and rarely will.

**Comment by oscar-b** [comment]
@f1multiviewer It would be nice if the default behavior would be to keep OBCs always synced to the main stream (if it's open). Like @loheiman says, it seems very unlikely that you'd ever want to watch them un-synced? 

For instance, having the main stream running and three OBCs, when switching one of them to another driver, I need to find the main window (running on my TV) click it so it has focus and then press S to sync. Having default sync from the main stream in this case would make it a lot easier switching OBCs.

One way could be to mark a stream as the "leader", and by default this is the first race feed you open. All other streams being opened will automatically sync to this. Pausing the "leader" stream should also pause the streams following this.

**Comment by rs-garrick** [comment]
That would be fantastic.  +1!  The last window you "Synced" should be the default leader.

**Comment by pbocsak** [comment]
I think you should be able to choose between two ways of operating

Option 1: You should be able to define either a specific window as Master (and for instance, right-click on a given window, then set it as "Master")

Option 2: You should toggle whether you want all windows to be synced. That way, if you wish to move the cursor backwards / forward on a given window (i.e., Data Channel on your 2nd monitor, in a setup where you're watching the Live Channel your main monitor, your main monitor should follow suit). This way, you'd be able to quickly scroll through a specific point in time, without losing the synchronicity across all your screens (my reasoning is that if everything is sync'd up, and you seek back 30 seconds on the Data Channel because you want to look at something specific that happened at a specific time, you will sooner or later "resume" watching it all the way you did before seeking back 30 seconds, which was all synced up, as you don't want the Data Channel to be lagging behind the "main" stream).

**Comment by argrithmag** [comment]
Since this is still opened, I also vote +1.  My main issue is if i get a phone call or something and pause a screen, I'd like it to pause all the screens, and if i hit play, bounce around, it keep all the other screens synced to that time.

**Comment by mattrazzo** [comment]
If the app was a set of a panels inside the same window (a different feature request), the "focus" issue would go away at a minimum.  But yes, everything should always be synced.

**Comment by captainprice42** [comment]
İs it forgetted?
